tests/e2e/client/apiproxy_bookstore_test.py

```diff
@@ -188,6 +188,20 @@ def verify_jwt_locations(self):
         })
         self.assertEqual(response.status_code, 200)
 
+    def run_options_queries(self):
+        # turn off verbose log
+        verbose = FLAGS.verbose
+        FLAGS.verbose = False
+        self.set_verbose(False)
+
+        print('Running CORS requests, no verifications')
+        for _ in range(100):
```

Comment:
Can we have accuracy test? A random idea is like repeatedly calling this `OPTIONS /shelves` until the new service config is applied.

Reply:
Prefer not to test accuracy, as we can have 2 tests running at the same time. They use the same service.

Reply:
You can still pass flag to differentiate long run jobs and presubmit jobs so that no overwritting in the same time.

These queries don't make sense to me if no any verification.

Reply:
I thought about this for some time. Even if we limit it to only periodic tests, it's not easy. 

The background process makes a rollout, but ESPv2 may take up to 2 minutes to pick up this new rollout and serve traffic on it. How can the background process communicate the expected value to the e2e client script? And how can we account for the potential 2 minute difference between a deploy and ESPv2 picking it up?

See [this comment](https://github.com/GoogleCloudPlatform/esp-v2/pull/273#discussion_r470897756) for why I don't think verifications are needed. Our integration tests verify the new rollout is picked up. The requests in e2e here are just for debugging, not verification.

Reply:
ACK. The test is not easy to setup because the config update cannot be updated right before verification.

We still need to make sure if the new config is picked up for ESPv2 finally or as Wayne said, by something else(like configId) . 
We don't need to reason the loading time. 

One extra question: why call 100 times without sleep? I can image most likely the response will be same. On the other hand, there is possibility these calls hit the same allowCors which give all 200 or 404, like  one complete test using 30min.

Reply:
Simplified the e2e test. Removed all cors related items, we don't need to verify anything.